### ISSUE TITLE
[N64] Mario no Photopie: Don't works due to missing SmartMedia support

### ISSUE THREAD

**Issue by Shideravan** [ares-emulator/ares#134]
![image](https://user-images.githubusercontent.com/12163923/131252017-512c5dde-89ea-47e8-8408-01fc3e3a1975.png)
ares e6948d2

**Comment by sp1187** [comment]
This game cartridge has a built-in SmartMedia memory card reader, which is not currently emulated by ares.

**Comment by Shideravan** [comment]
Yes, you're right. The message says about the SmartMedia device, I changed the description.

**Comment by Shideravan** [comment]
It seems there's a patch for it runs without a SmartMedia card.
https://www.vice.com/en/article/8x8bjv/this-forgotten-nintendo-64-game-is-one-part-photoshop-one-part-mario-paint

**Comment by Shideravan** [comment]
There's a patch to run it without a SmartMedia card. It runs fine in ares (apart of performance issues, of course)
![image](https://user-images.githubusercontent.com/12163923/131260932-e743dab5-b33d-4885-8b4c-07d49e8f449a.png)
